### DISCUSSION TITLE
What substantial differences are there between this crate and `chrono`?

### DISCUSSION THREAD

**Discussion by frederikhors** [time-rs/time#659] [Question] [answered]
I'm new to Rust and I'm wondering: what substantial differences are there between this crate and [chrono](https://crates.io/crates/chrono)?

**Comment by jhpratt** [comment, marked as answer]
See #308. I have never gotten around to creating a full comparison, as my priorities are elsewhere. I'm also naturally not unbiased, so a comparison would necessarily be better from a third party (albeit with checking for correctness from the maintainers).
